mailpoet/tests/unit/Automation/Engine/Validation/AutomationRules/ValidStepOrderRuleTest.php

```
@@ -230,6 +230,10 @@ public function getSubjectKeys(): array {
 
       public function validate(StepValidationArgs $args): void {
       }
+
+      public function onDuplicate(\MailPoet\Automation\Engine\Integration\Step $step, \MailPoet\Automation\Engine\Data\Automation $automation): void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$step'. (Unused Code Rules)

(UnusedFormalParameter)